torch/_torch_docs.py

```diff
@@ -6342,24 +5015,11 @@ def parse_kwargs(desc):
 
     >>> A = torch.randn(3, 3)
     >>> torch.det(A)
-
-    -0.3534
-    [torch.FloatTensor of size ()]
-
+    tensor(0.6231)
     >>> torch.logdet(A)
-
-    nan
-    [torch.FloatTensor of size ()]
-
+    tensor(-0.4730)
```

Comment:
It'd be nice if we can generate one that has negative `det`... The original purpose was to show that `slogdet` can handle negative `det`.

Reply:
We're making doc changes to li-roy/pytorch:tensorprintdocs -- feel free to push changes!